frontend/common/src/constants/localizedConstants.tsx

```diff
@@ -396,6 +396,43 @@ export const getEducationType = (
     `Invalid educationType ${educationTypeId}`,
   );
 
+export const operationalRequirementsBolded = defineMessages({
```

Comment:
This looks the same as the regular messages.  I think you abandoned this but forgot to delete it?

Reply:
That one pertains to this bit in the PR post. 

"As well, adjust WorkPreferences to have bolded text in the first question
For that component, it does not match up with the wireframe
https://www.figma.com/proto/PrvXEDac0HVp7dF7SLWVFm/Candidate-profile-interfaces---V5.0.0?node-id=2485%3A5000&starting-point-node-id=1%3A2
So not sure what text needs to be bolded in the second question"

I geared up to export a bunch of bolded things in LocalizedConstants but realized there was a discrepancy between the component and Wireframe. 
I am going to assume then that I do not need to add any bolding to the second question

Reply:
Time to track down a designer, I guess. 😁

Reply:
@Jerryescandon 
We've kind of ended up in an odd spot, wireframe and component are a bit different.
Wireframe is like so
https://www.figma.com/proto/PrvXEDac0HVp7dF7SLWVFm/Candidate-profile-interfaces---V5.0.0?node-id=2485%3A5000&starting-point-node-id=1%3A2
Component is like so
![image](https://user-images.githubusercontent.com/40485260/163030733-d4af10e4-8f4f-465b-a477-cedff7634900.png)

Is there anything you'd like to be done to the component, like any formatting changes? As is, it is rendering the checklist straight from the database formatting, so outputting entries how they are stored in database

Reply:
There is another thing I could try, so please don't feel rushed to answer soon @Jerryescandon

Reply:
sorry, I think it might be impossible to add styling (bolding) onto the checklist things 😓

Reply:
Assuming the text gets updated (discarded?) through issue #2212, I'm less worried about bold or not bold. If bolding is not easy using the current component then leave it plain. The copy however should definitely match the wireframe, since there's some behavior impacts we are trying to achieve by wording the options like that.

Reply:
No worries about the bold. Drop it like it's hot.

Reply:
Aye aye! Thank you very much!
Text will be adjusted as requested

Reply:
I took a look at the code and I think it's simple to get the bolding.  You already have the formatting function in the component so just add the markup to the localize strings and hook it into the formatMessage function.
![image](https://user-images.githubusercontent.com/8978655/163050833-db7c7673-3737-4e42-8b38-ce5e4826fdb8.png)
![image](https://user-images.githubusercontent.com/8978655/163050882-72eb62a8-0a53-4ff0-a42a-98c9126c9007.png)

Two more things:
- Double-check the wireframe - some strings have parenthetical parts like _(Occasionally)_
- `OperationalRequirementWireframe` is not a very helpful name to describe the use - for candidates (as opposed to hiring managers).  Perhaps something like OperationalRequirementCandidateDescription?  It's long but that's what autocomplete is for. :grin: Maybe @Jerryescandon has a better idea.  Nomenclature is hard!

Reply:
@petertgiles 
Oh, neat idea. I was trying to pass in the bold function from LocalizedConstants 😅 
Unfortunately, it doesn't appear to have worked, I've pushed up the attempt.

Reply:
> it doesn't appear to have worked

You're 99.99% of the way there. Your formatting function is `boldText` but your markup is `bold`.  :blush:

Reply:
> > it doesn't appear to have worked
> 
> You're 99.99% of the way there. Your formatting function is `boldText` but your markup is `bold`. 😊

😭 
Alright, fixed that